qutebrowser/misc/miscwidgets.py

```diff
@@ -96,6 +97,22 @@ def __on_cursor_position_changed(self, _old, new):
         if new < self._promptlen:
             self.setCursorPosition(self._promptlen)
 
+    @pyqtSlot()
+    def __on_selection_changed(self):
+        """Prevent the prompt from being selected.
+
+        We use __ here to avoid accidentally overriding it in subclasses.
+        """
+        if (not self.hasSelectedText() or
+                self.selectionStart() >= self._promptlen):
+            return
+        selectionEnd = self.selectionEnd()
```

Comment:
selectionEnd was added in qt 5.10, and causes:

```
Traceback (most recent call last):
  File "/home/jay/Code/qutebrowser/qutebrowser/misc/miscwidgets.py", line 109, in __on_selection_changed
    selectionEnd = self.selectionEnd()
AttributeError: 'Command' object has no attribute 'selectionEnd'
```

the cursorForward method does work on older versions though, but dosen't help when selecting everything. FWIW though, I liked the previous behavior of selecting the `:` too, as I could copy ex commands into IRC pretty easily using it. On the downside typing dosen't work after a triple click.

Reply:
I think that would be a good idea to implement. Closing this FR because that one is better.

About selecting `:` with triple click -- it's up to you, but it would be better to explicitly specify the behavior so people know that it's a feature, not a bug.